packages/libs/eda/src/lib/core/components/visualizations/implementations/NetworkVisualization.tsx

```diff
@@ -247,6 +263,18 @@ function NetworkViz(props: VisualizationProps<Options>) {
       };
     });
 
+    // update selected node label
+    const labels =
```

Comment:
can we change this to `nodeLabels` just for clarity, please?

Reply:
@asizemore changed to `nodeLabels` as suggested